easybuild/easyconfigs/a/alleleCount/alleleCount-4.2.1-GCC-11.3.0.eb

```diff
@@ -0,0 +1,50 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Institute, London, UK
+##
+
+easyblock = 'CmdCp'
```

Comment:
Can you look into using `MakeCp` or `ConfigureMake` instead, so we can avoid running the `setup.sh` which still downloads a bunch of stuff?